storage/key_index.go

```diff
@@ -132,6 +132,41 @@ func (ki *keyIndex) get(atRev int64) (modified, created revision, ver int64, err
 	return revision{}, revision{}, 0, ErrRevisionNotFound
 }
 
+// since returns recorded valid revisions since the given rev.
```

Comment:
since returns revisions since the give rev.
Only the revision with the largest sub revision will be returned if multiple revisions have the same main revision.